plugin-server/tests/main/ingestion-queues/session-recording/session-recordings-consumer.test.ts

```diff
@@ -648,4 +657,20 @@ describe('ingester', () => {
             ])
         })
     })
+
+    describe('when a team is disabled', () => {
+        it('can commit even if an entire batch is disabled', async () => {
```

Comment:
the interesting thing given what we now now is that the setup here was wrong and this test _should_ have been failing

Reply:
Probably us testing something that is different to reality. The test worked because the "lastKnownCommit" gets correctly setup. It could be that real events don't quite behave in the way the test does